.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,93 @@
+name: Build and upload to PyPI
+
+# Build on every branch push, tag push, and pull request change:
+on: push
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 2
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.6'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.4.2
+
+      - name: Cache OpenCV dependency for macOS
+        if: ${{ runner.os == 'macOS' }}
```

Comment:
How does OpenCV get built and cached in the Ubuntu environment?

Reply:
There is no need for a cache in ubuntu because we have it built into the container image.

Reply:
👍

Reply:
Yep, makes sense, I see `wildme/manylinux:latest` being used as the build context for `cibuildwheel`